src/modules/rolle/repo/rolle.repo.ts

```diff
@@ -71,6 +71,15 @@ export class RolleRepo {
         return rolle && mapEntityToAggregate(rolle);
     }
 
+    public async findByName(searchStr: string): Promise<Option<Rolle<true>[]>> {
+        const rollen: Option<RolleEntity[]> = await this.em.find(
+            this.entityName,
+            { name: { $ilike: '%' + searchStr + '%' } },
+            { populate: ['merkmale', 'systemrechte'] as const },
```

Comment:
Include a limit here to not load all entities?

Reply:
is currently a constant, will be parameterized

Reply:
limit is now used from the incoming Find-parameters